adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleRewardedAd.m

```diff
@@ -92,6 +85,13 @@ - (void)loadRewardedAd {
   VungleAdsExtras *extras = [[VungleAdsExtras alloc] init];
   [extras setWithWatermark:[_adConfiguration.watermark base64EncodedStringWithOptions:0]];
   [_rewardedAd setWithExtras:extras];
+    if ([_adConfiguration extras] &&
+        [[_adConfiguration extras] isKindOfClass:[VungleAdNetworkExtras class]]) {
+      VungleAdNetworkExtras *extras = (VungleAdNetworkExtras *)[_adConfiguration extras];
```

Comment:
I think this should be named networkExtras or should be refactored into a separate function/method like `findPlacement:`. One can get confused with extras declared at line 92/85.

Reply:
Ok. Updated